static/js/components/ErrorMessage.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import LoginButton from '../containers/LoginButton';
+import { Alert } from 'react-bootstrap';
+
+
+class ErrorMessage extends React.Component {
+  static propTypes = {
+    errorInfo:   React.PropTypes.object.isRequired,
+  };
+
+  state = {
+    alertVisible: true
+  };
+
+  render() {
+    if (!this.state.alertVisible) {
+      return null;
+    }
+
+    const {errorInfo} = this.props;
+    if (errorInfo.errorStatusCode === undefined) {
+      return null;
+    }
+    let errorCode, userMessage = '';
+    if (errorInfo.error_code !== undefined) {
+      errorCode = errorInfo.error_code + ': ';
+    }
+    if (errorInfo.user_message !== undefined) {
+      userMessage = 'Additional info: ' + errorInfo.user_message;
```

Comment:
We are using template literals instead of string concatenation, so this would be `` `Additional info: ${errorInfo.user_message}` ``